### PR TITLE
Add documentation on Postman Integration

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

**Comment by RukshiW** [comment]
Corrects https://github.com/wso2-enterprise/choreo/issues/3894

File: en/docs/references/postman-integration.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Postman Integration
```
Comment (Nashaath) [inline en/docs/references/postman-integration.md:1]: Should we consider changing this to something like "Test Your Service Using Postman" or something along those lines?  I feel this title can cause confusion.

Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Changed

File: en/mkdocs.yml [hunk 2]
```diff
@@ -69,6 +69,7 @@ nav:
           - Ballerina Expression Syntax: references/ballerina-expression-syntax.md
           - Anonymous Access: references/anonymous-access.md
           - Data Mapping: references/data-mapping.md
+          - Postman Integration: references/postman-integration.md
```
Comment (Nashaath) [inline en/mkdocs.yml:72]: Need to change this also to reflect the title change.

File: en/docs/references/postman-integration.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+In order to use the Postman Editor:
+
+- Make sure you have logged into the Choreo console and created a service. For instructions to create a service, see [Create Your First Service](../services/create-your-first-service.md).
+
+    !!! tip
+        This guide uses the service created in [Create Your First Service](../services/create-your-first-service.md) as an example.
+        
+- Have an active Postman account.
+
+## Step 1: Generate Postman API key
```
Comment (Nashaath) [inline en/docs/references/postman-integration.md:16]: Should we change the steps also to go with the change in title if it makes sense. 

For example
Step 1: Connect your Postman account to Choreo
Step 2:

Comment (RukshiW) [reply en/docs/references/postman-integration.md:16]: A separate Postman key ios generated. for each service. Therefore, I think "Connect your Postman account to Choreo" does not give the right idea.

File: en/docs/references/postman-integration.md (hunk 1 above)
Comment (anupama-pathirage) [reply en/docs/references/postman-integration.md]: Common best practice is to use the file name similar to the topic. i.e. `test-your -service-using-postman.md`

File: en/docs/references/postman-integration.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+To generate the Postman API key so that you can connect your Postman account to Choreo, follow this procedure:
+
+1. Sign in to your Postman account.
+
+2. Go to your profile, and click **Account Settings**. Then click **Postman API Keys**
+
+    ![Generate Postman API key](../assets/img/postman/generate-postman-api-key.png){.cInlineImage-full}
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:24]: Isn't `{.cInlineImage-half}` more suitable for this?

Comment (anupama-pathirage) [reply en/docs/references/postman-integration.md:24]: Also is this image really useful to the user? If we really need to have it get the screenshot after selecting `Postman API Keys` instead of above.

image

File: en/docs/references/postman-integration.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+3. Click **Generate API Key**.
+
+4. In the **Generate an API key** dialog box, enter `choreo` as the key name. Then click **Generate API Key**.
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:28]: It is not mandatory to name it as `choreo`. So shouldn't we reword this to say enter the key name , e.g. `choreo` or something similar to that?

File: en/docs/references/postman-integration.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+5. Copy the key that is generated and save it in a safe location.
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:30]: IMO, `and save it in a safe location.` is not needed.

File: en/docs/references/postman-integration.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+2. Go to the **Services** card and click **Explore**. Then click on the service you want to test.
+
+3. Once the service opens, click on the **Test** icon in the left pane. Then click on the **Postman** icon.
+
+    ![Access Postman test view](../assets/img/postman/access-postman-test-view.png){.cInlineImage-full}
+    
+    As shown in the image above, the **Postman** panel opens to the right of the low-code diagram with instructions to integrate Postman. You have already completed the first two steps given there.
+
+4. Click on the link provided in Step 3 in the **Postman** panel.
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:46]:
```suggestion
4. Click on the link provided in the step 3 in the **Postman** panel.
```

File: en/docs/references/postman-integration.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+
+Now you can proceed to create a new workspace to link to your service or select an existing workspace in your Postman account.
+
+## Step 3: Link a workspace to your service
+
+In this step, let's connect a workspace to your service by following this procedure:
+
+1. In the **Postman** right panel that you opened, expand the **Select or create a workspace** field.
+
+    ![Postman right panel](../assets/img/postman/postman-right-panel.png){.cInlineImage-full}
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:66]: This should be styled as half image.

File: en/docs/references/postman-integration.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+    | **Description** | `Test the covid-Status service` |
+    
+4. Click **Create & Link**.
+
+Now you have linked your service to the new workspace named `covid-Status`.
+
+## Step 4: Create a collection
+
+!!! tip
+    If you linked an existing workspace to the service and that workspace has collections that are already created in it, you can skip this step.
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:86]: This sentence is bit unclear, consiter rewording, May be having two `that` words is one problem.

File: en/docs/references/postman-integration.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+    Your API key was validated by Choreo
+    ```
+   
+6. Click **Save**. 
+
+Now you can proceed to create a new workspace to link to your service or select an existing workspace in your Postman account.
+
+## Step 3: Link a workspace to your service
+
+In this step, let's connect a workspace to your service by following this procedure:
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:62]: Instead of directly asking to create workspace, we better say what is workspace and why we need to create it  in 1-2 sentences. Otherwise, it is like we are asking to follow blindly without understanding the reasoning.

File: en/docs/references/postman-integration.md [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+    | **Field**       | **Value**                       |
+    |-----------------|---------------------------------|
+    | **Name**        | `covid-Status`                  |
+    | **Description** | `Test the covid-Status service` |
+    
+4. Click **Create & Link**.
+
+Now you have linked your service to the new workspace named `covid-Status`.
+
+## Step 4: Create a collection
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:83]: Instead of directly asking to create a collection, we better say what is collection and why we need to create it  in 1-2 sentences. Otherwise, it is like we are asking to follow blindly without understanding the reasoning.

File: en/docs/references/postman-integration.md [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+    
+To create a collection, follow this procedure:
+
+1. Under **Collections** in the **Postman** right panel, click **+ Add**.
+
+2. Enter a name and a description. For this example, let's enter them as follows:
+
+    | **Field**       | **Value**                             |
+    |-----------------|---------------------------------------|
+    | **Name**        | `collection1`                         |
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:96]: This is not a good value.

File: en/docs/references/postman-integration.md [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+    | **Description** | `Collection for covid-status service` |
+    
+3. Click **Create**.
+
+As a result, the collection is created and displayed under **Collections** in the **Postman** right panel.
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:101]: Shouldn't we say just `Postman panel` instead of `Postman right panel`?

File: en/docs/references/postman-integration.md [hunk 14]
```diff
@@ -0,0 +1,129 @@
...
+
+## Step 5: Create a request
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:103]: Add a introductory sentence to say now u are done with test creation and lets test this or something similar.

File: en/docs/references/postman-integration.md [hunk 15]
```diff
@@ -0,0 +1,129 @@
...
+
+To create a request, follow this procedure:
+
+1. In the **Postman** right panel, click on the collection that you created. This opens the **Requests** section.
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:107]: It is not on the collection, we need to say click on the link icon on the collection.

File: en/docs/references/postman-integration.md [hunk 16]
```diff
@@ -0,0 +1,129 @@
...
+
+2. Click **+ Add**.
+
+3. Enter a request name (e.g., `test1`)
+
+4. Select the required HTTP method. You can also add the query parameters, the header, and the request body as required.
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:113]:
```suggestion
4. Select the required HTTP method. You can also add the query parameters, headers, and the request body as required.
```

File: en/docs/references/postman-integration.md [hunk 17]
```diff
@@ -0,0 +1,129 @@
...
+
+## Step 5: Create a request
+
+To create a request, follow this procedure:
+
+1. In the **Postman** right panel, click on the collection that you created. This opens the **Requests** section.
+
+2. Click **+ Add**.
+
+3. Enter a request name (e.g., `test1`)
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:111]: Not a meaningful name

File: en/docs/references/postman-integration.md [hunk 18]
```diff
@@ -0,0 +1,129 @@
...
+
+4. Select the required HTTP method. You can also add the query parameters, the header, and the request body as required.
+
+    In this example, you can select **GET** as the HTTP method and add `country` as a query parameter.
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:115]: Why adding a query parameter without a value here?

File: en/docs/references/postman-integration.md [hunk 19]
```diff
@@ -0,0 +1,129 @@
...
+    
+    ![Postman request](../assets/img/postman/postman-request.png){.cInlineImage-full}
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:117]: Hide out your name in the image. This should be half image. Choose size of the image intentionally, this issue is repetitive, so check all the places.

File: en/docs/references/postman-integration.md [hunk 20]
```diff
@@ -0,0 +1,129 @@
...
+    
+5. Click **Save**.
+
+## Step 6: Check your Postman account
+
+Sign in to your Postman account and check your workspaces. The workspace you created via Choreo will be available as shown below.
+
+![View workspace in Postman](../assets/img/postman/workspace-in-postman.png){.cInlineImage-full}
+
+Click on the workspace. You can view the collection and the request as follows.
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:127]: The documentation is abruptly ended without saying how to test  by sending the request.

File: en/docs/references/postman-integration.md [hunk 21]
```diff
@@ -0,0 +1,129 @@
...
+    
+    ![Postman request](../assets/img/postman/postman-request.png){.cInlineImage-full}
+    
+5. Click **Save**.
+
+## Step 6: Check your Postman account
+
+Sign in to your Postman account and check your workspaces. The workspace you created via Choreo will be available as shown below.
+
+![View workspace in Postman](../assets/img/postman/workspace-in-postman.png){.cInlineImage-full}
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:125]: This should be half image

File: en/docs/references/postman-integration.md [hunk 22]
```diff
@@ -0,0 +1,129 @@
...
+
+5. Copy the key that is generated and save it in a safe location.
+
+## Step 2: Add Postman API key to your service
+
+In this step, you will be connecting your service to your Postman account by adding the Postman API key that you generated to your service.
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
+
+2. Go to the **Services** card and click **Explore**. Then click on the service you want to test.
```
Comment (anupama-pathirage) [inline en/docs/references/postman-integration.md:38]: Since this is step by step guide, we should say we are using our covid example for this guide.

File: en/docs/references/postman-integration.md (hunk 18 above)
Comment (anupama-pathirage) [reply en/docs/references/postman-integration.md:115]: Also make it a working guide. For covid example to get it working we need to add `path` too as `stats/`

File: en/docs/references/postman-integration.md (hunk 1 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Renamed file.

File: en/mkdocs.yml (hunk 2 above)
Comment (RukshiW) [reply en/mkdocs.yml]: Changed navigator title

File: en/docs/references/postman-integration.md (hunk 4 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Removed the image.

File: en/docs/references/postman-integration.md (hunk 6 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Removed this phrase.

File: en/docs/references/postman-integration.md (hunk 7 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Changed

File: en/docs/references/postman-integration.md (hunk 8 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Corrected

File: en/docs/references/postman-integration.md (hunk 9 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Rephrased this.

File: en/docs/references/postman-integration.md (hunk 10 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Added a brief description and provided a link for further information.

File: en/docs/references/postman-integration.md (hunk 11 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Added a brief description and provided a link for further information.

File: en/docs/references/postman-integration.md (hunk 12 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Changed the sample value.

File: en/docs/references/postman-integration.md (hunk 13 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Corrected.

File: en/docs/references/postman-integration.md (hunk 14 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Added an introductory sentence.

File: en/docs/references/postman-integration.md (hunk 15 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Corrected.

File: en/docs/references/postman-integration.md (hunk 16 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Corrected.

File: en/docs/references/postman-integration.md (hunk 17 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Changed the sample name.

File: en/docs/references/postman-integration.md (hunk 18 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Corrected.

File: en/docs/references/postman-integration.md (hunk 19 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Corrected.

File: en/docs/references/postman-integration.md (hunk 20 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Added more steps to send a request.

File: en/docs/references/postman-integration.md (hunk 22 above)
Comment (RukshiW) [reply en/docs/references/postman-integration.md]: Done

File: en/docs/references/test-your-service-via-postman.md [hunk 23]
```diff
@@ -0,0 +1,145 @@
...
+
+In this step, you will be connecting your service to your Postman account by adding the Postman API key that you generated to your service.
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
+
+2. Go to the **Services** card and click **Explore**. Then click on the service you want to test (in this example, **World Bank Data to COVID-19 Statistics**).
+
+3. Once the service opens, click on the **Test** icon in the left pane. Then click on the **Postman** icon.
+
+    ![Access Postman test view](../assets/img/postman/access-postman-test-view.png){.cInlineImage-full}
```
Comment (anupama-pathirage) [inline en/docs/references/test-your-service-via-postman.md:40]: This image has your name.

Comment (RukshiW) [reply en/docs/references/test-your-service-via-postman.md:40]: Hid the name.

File: en/docs/references/test-your-service-via-postman.md [hunk 24]
```diff
@@ -0,0 +1,145 @@
+# Test Your Service via Postman
+
+The Postman Editor allows you to integrate with your Postman account to easily test your services.
```
Comment (Nashaath) [inline en/docs/references/test-your-service-via-postman.md:3]: I feel that users. will not understand what "Postman Editor" is here.
Shall we improve this description to give the idea that Choreo allows users to test their services using Postman.

File: en/docs/references/test-your-service-via-postman.md [hunk 25]
```diff
@@ -0,0 +1,145 @@
...
+    
+3. In the request path displayed, replace `{country}` with `USA` as shown below:
+
+    ![Update request path](../assets/img/postman/update-request-path.png){.cInlineImage-full}
+    
+4. Click **Send**.
+
+    The response is logged as shown below.
+
+    ![Response](../assets/img/postman/response.png){.cInlineImage-full}
```
Comment (anupama-pathirage) [inline en/docs/references/test-your-service-via-postman.md:145]: Since this looks like a step by step guide don't we need to have ending sentence saying you have completed xxx.

File: en/docs/references/test-your-service-via-postman.md [hunk 26]
```diff
@@ -0,0 +1,145 @@
...
+        
+- Have an active Postman account.
+
+## Step 1: Generate Postman API key
+
+To generate the Postman API key so that you can connect your Postman account to Choreo, follow this procedure:
+
+1. Sign in to your Postman account.
+
+2. Go to your profile, and click **Account Settings**. Then click **Postman API Keys**
```
Comment (anupama-pathirage) [inline en/docs/references/test-your-service-via-postman.md:22]:
```suggestion
2. Go to your profile, and click **Account Settings**. Then click **Postman API Keys**.
```

File: en/docs/references/test-your-service-via-postman.md [hunk 27]
```diff
@@ -0,0 +1,145 @@
...
+    Your API key was validated by Choreo
+    ```
+   
+6. Click **Save**. 
+
+You can now create a new workspace to link to your service or select an existing workspace in your Postman account.
+
+## Step 3: Link a workspace to your service
+
+In this step, let's connect a workspace to your service. You can use workspaces in Postman to manage your Postman components for different tasks. In this example, you can create a workspace to have the collections you create for testing the `World Bank Data to COVID-19 Statistics` service. For more information about workspaces, see [Postman Documentation - Using and managing workspaces](https://learning.postman.com/docs/collaborating-in-postman/using-workspaces/managing-workspaces/).
```
Comment (anupama-pathirage) [inline en/docs/references/test-your-service-via-postman.md:60]: To refer `World Bank Data to COVID-19 Statistics` step 2.2 bold text is used. Use a consistent way.

File: en/docs/references/test-your-service-via-postman.md [hunk 28]
```diff
@@ -0,0 +1,145 @@
...
+    | **Name**        | `covid-Status`                  |
+    | **Description** | `Test the covid-Status service` |
+    
+4. Click **Create & Link**.
+
+Now you have linked your service to the new workspace named `covid-Status`.
+
+## Step 4: Create a collection
+
+In this step, you will be creating a collection. The collection runner in Postman runs multiple requests in a specified sequence and logs the results. Each collection represents one request. For more information about collections, see [Postman Documentation - Collection Runner](https://learning.postman.com/docs/running-collections/intro-to-collection-runs/)
```
Comment (anupama-pathirage) [inline en/docs/references/test-your-service-via-postman.md:85]:
```suggestion
In this step, you will be creating a collection. The collection runner in Postman runs multiple requests in a specified sequence and logs the results. Each collection represents one request. For more information about collections, see [Postman Documentation - Collection Runner](https://learning.postman.com/docs/running-collections/intro-to-collection-runs/).
```

File: en/docs/references/test-your-service-via-postman.md [hunk 29]
```diff
@@ -0,0 +1,145 @@
...
+    
+To create a collection, follow this procedure:
+
+1. Under **Collections** in the **Postman** panel, click **+ Add**.
+
+2. Enter a name and a description. For this example, let's enter them as follows:
+
+    | **Field**       | **Value**                             |
+    |-----------------|---------------------------------------|
+    | **Name**        | `invoke-covid-status service`         |
```
Comment (anupama-pathirage) [inline en/docs/references/test-your-service-via-postman.md:98]: Why we use `-` for the first 3 words , but not for the last one?

File: en/docs/references/test-your-service-via-postman.md [hunk 30]
```diff
@@ -0,0 +1,145 @@
...
+    
+    As shown above, the drop-down list displays the existing workspaces in your Postman account. You can link your service to an existing workspace or create a new one. In this example, let's create a new workspace.
+    
+2. Click **+ Create New**.
+
+3. Enter a name and a description for the new workspace. For this example, let's enter them as follows:
+
+    | **Field**       | **Value**                       |
+    |-----------------|---------------------------------|
+    | **Name**        | `covid-Status`                  |
```
Comment (anupama-pathirage) [inline en/docs/references/test-your-service-via-postman.md:76]: Is having upper case `S` is intentiona? What is the best practice to follow?

File: en/docs/references/test-your-service-via-postman.md [hunk 31]
```diff
@@ -0,0 +1,145 @@
...
+
+In this step, let's create a request that you can send via Postman to invoke your service
+
+To create a request, follow this procedure:
+
+1. In the **Postman** panel, click on the **>** icon of the collection that you created. This opens the **Requests** section.
+
+2. Click **+ Add**.
+
+3. Enter a request name (e.g., `covid-status-request-1`)
```
Comment (anupama-pathirage) [inline en/docs/references/test-your-service-via-postman.md:115]: To me names are inconsistent. How about having names as:
`COVID Service Workspace`
`COVID Service Test Collection`
`COVID Country Status Request`

File: en/docs/references/test-your-service-via-postman.md [hunk 32]
```diff
@@ -0,0 +1,145 @@
...
+    | **Name**        | `invoke-covid-status service`         |
+    | **Description** | `Collection for covid-status service` |
+    
+3. Click **Create**.
+
+As a result, the collection is created and displayed under **Collections** in the **Postman** panel.
+
+## Step 5: Create a request
+
+In this step, let's create a request that you can send via Postman to invoke your service
```
Comment (anupama-pathirage) [inline en/docs/references/test-your-service-via-postman.md:107]:
```suggestion
In this step, let's create a request that you can send via Postman to invoke your service.
```

File: en/docs/references/test-your-service-via-postman.md (hunk 31 above)
Comment (anupama-pathirage) [reply en/docs/references/test-your-service-via-postman.md:115]:
```suggestion
3. Enter a request name (e.g., `covid-status-request-1`).
```

**Comment by anupama-pathirage** [comment]
@RukshiW Please fix the file conflict too

File: en/docs/references/test-your-service-via-postman.md (hunk 29 above)
Comment (RukshiW) [reply en/docs/references/test-your-service-via-postman.md]: Replaced sample values

File: en/docs/references/test-your-service-via-postman.md (hunk 30 above)
Comment (RukshiW) [reply en/docs/references/test-your-service-via-postman.md]: Replaced sample values.

File: en/docs/references/test-your-service-via-postman.md (hunk 27 above)
Comment (RukshiW) [reply en/docs/references/test-your-service-via-postman.md]: Corrected.

File: en/docs/references/test-your-service-via-postman.md (hunk 25 above)
Comment (RukshiW) [reply en/docs/references/test-your-service-via-postman.md]: Added a closing sentence.

File: en/docs/references/test-your-service-via-postman.md (hunk 24 above)
Comment (RukshiW) [reply en/docs/references/test-your-service-via-postman.md]: Updated.

**Comment by anupama-pathirage** [comment]
@RukshiW Please remove unnecessary generate-postman-api-key.png  and .DS_Store  files.

File: en/mkdocs.yml [hunk 33]
```diff
@@ -69,6 +69,7 @@ nav:
           - Ballerina Expression Syntax: references/ballerina-expression-syntax.md
           - Anonymous Access: references/anonymous-access.md
           - Data Mapping: references/data-mapping.md
+          - Test Your Service via Postman: references/postman-integration.md
```
Comment (Nashaath) [inline en/mkdocs.yml:72]: Need to change the filename here.

**Comment by RukshiW** [comment]
Deleted unnecessary files

File: en/docs/references/test-your-service-via-postman.md [hunk 34]
```diff
@@ -0,0 +1,147 @@
+# Test Your Service via Postman
+
+This guide explains how to test your services by connecting them to your Postman account via an API key.
+
+## Prerequisites
+
+To use the Postman Editor:
+
+- Make sure you have logged into the Choreo console and created a service. For instructions to create a service, see [Create Your First Service](../services/create-your-first-service.md).
```
Comment (Nashaath) [inline en/docs/references/test-your-service-via-postman.md:9]: Since we've used "sign in" to Choreo Console in almost all other. places, shall we use the same here too?
Also need to use capital C for Console.

**Review by Nashaath — APPROVED** [review]

**Review by anupama-pathirage — APPROVED** [review]
